address_to_census_tract.py

```diff
@@ -4,84 +4,194 @@
 associated census tract within Washington.
 
 To run:
-    ./address_to_census_tract.py
+    ./address_to_census_tract.py {filepath}
+
+Examples:
+    ./address_to_census_tract.py data/test/testset.json
+    ./address_to_census_tract.py data/test/testset.csv 
 
 Requirements:
-    geocoder
     shapely
+    pandas
+    smartystreets_python_sdk
 """
-import geocoder
 import json
+import os
+from sys import argv
+from textwrap import dedent
 from shapely.geometry import shape, Point
-from sys import stderr
+import logging
+import pandas as pd
+from smartystreets_python_sdk import StaticCredentials, exceptions, ClientBuilder
+from smartystreets_python_sdk.us_street import Lookup
+from smartystreets_python_sdk.us_extract import Lookup as ExtractLookup
+
+LOG = logging.getLogger(__name__)
 
 
 def main():
-    with open("data/test/testset.json", encoding = "UTF-8") as file:
+    if argv[1].endswith('.json'):
+        process_json(argv[1])
+    elif argv[1].endswith('.csv'):
+        process_csv(argv[1])
```

Comment:
The number of args should be checked before accessing `argv[1]`. Perhaps we should switch to using argparse now.